segmentation/utils/spatial_analysis.py

```diff
@@ -0,0 +1,259 @@
+import pandas as pd
+import xarray as xr
+import numpy as np
+import scipy
+import statsmodels
+from statsmodels.stats.multitest import multipletests
+
+# Erin's Data Inputs
+
+# cell_array = pd.read_csv("/Users/jaiveersingh/Downloads/SpatialEn"
+#                          "richment/granA_cellpheno_CS-asinh-norm_revised.csv")
+# marker_thresholds = pd.read_csv("/Users/jaiveersingh/Downloads/Sp"
+#                                 "atialEnrichment/markerThresholds.csv")
+# dist_matrix = np.asarray(pd.read_csv("/Users/jaiveersingh/Documen"
+#                                      "ts/MATLAB/distancesMat5.csv",
+#                                      header=None))
+
+
+def compute_close_cell_num(patient_data_markers, thresh_vec,
```

Comment:
This isn't a main function, so now that we're happy with it lets move this to spatial_analysis_utils